core/tests/mad_hatter/test_mad_hatter.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from cat.mad_hatter.mad_hatter import MadHatter
+from cat.looking_glass.cheshire_cat import CheshireCat
+
+
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+def test_install_plugin():
+    ccat = CheshireCat()
+    ccat.get_plugin_path = MagicMock(return_value="./tests/mad_hatter/plugin_folder")
+    ccat.bootstrap = MagicMock()
+    mad_hatter = MadHatter(ccat)
+
+    with patch('importlib.import_module'):
+        mad_hatter.install_plugin("./tests/mad_hatter/plugin_test.zip")
```

Comment:
Please no zip files in the repo, create the mock plugin zip at runtime as done [here](https://github.com/cheshire-cat-ai/core/blob/main/core/tests/routes/test_routes_plugins.py#L42) (using the Zip class maybe)

It is useful so we can commit on the [mock plugin](https://github.com/cheshire-cat-ai/core/tree/main/core/tests/mocks/mock_plugin) sources